src/components/data/WdsTroubleshooter.js

```diff
@@ -75,71 +75,58 @@ export const WdsTroubleshooter = ({ onDismiss, workspaceId, mrgId }) => {
     ['running', () => icon('loadingSpinner', { size, style: { color: colors.primary() }, 'aria-label': 'Validation Running' })]
   )
 
-  const clippy = (label, value) => !!value && h(Fragment, [value, h(ClipboardButton, {
-    'aria-label': `Copy ${label} to clipboard`,
-    style: { marginLeft: '1rem' },
-    text: value
-  })])
-
-  const troubleShooterRow = ([label, text, iconRunning, iconSuccess, copy, element]) => {
+  const troubleShooterRow = ([label, text, iconRunning, iconSuccess, element]) => {
     return tr([
       td({ style: { fontWeight: 'bold' } }, [
         iconRunning ? checkIcon('running') : (iconSuccess ? checkIcon('success') : checkIcon('failure'))
       ]),
       td({ style: { fontWeight: 'bold' } }, [label]),
       td({ style: { fontWeight: 'bold' } }, [!!element]),
-      Utils.cond([!!element, () => element], () => Utils.cond([copy, () => td([clippy(label, text)])], () => td([h(Fragment, [text])]))
+      Utils.cond([!!element, () => element], () => td([h(Fragment, [text])])
       )
     ])
   }
 
-  // The proxyUrl is long and should be truncated, but
-  // we still want to be able to copy it, so it gets it own special element
-  // that doesn't use the clippy() helper
+  // The proxyUrl is long and should be truncated,
+  // so it gets it own special element
```

Comment:
How come we no longer want to enable the ability to copy?

Seems the URL is still truncated?

Reply:
sounds good!